test/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap

```diff
@@ -3592,7 +3592,7 @@ Array [
   Array [
     "notify",
     Object {
-      "body": "You have successfully undelegated 10 STAKEs.",
+      "body": "You have successfully undelegated 10 undefineds.",
```

Comment:
I think this should show a denom

Reply:
damn true

Reply:
uhh ? On my local file it shows the correct STAKE. but when I try to push it says `Everything up-to-date`

Reply:
updated